Campus-iOS/Map/Cafeterias/MealPlanViewModel.swift

```diff
@@ -21,6 +21,8 @@ final class MealPlanViewModel: ObservableObject {
     init(cafeteria: Cafeteria) {
         self.cafeteria = cafeteria
         self.title = cafeteria.name
+        
+        fetch()
     }
     
     func fetch() {
```

Comment:
Also make sure of some error handling here if this fetch request can't be done (eg. no network connection) -> Display some kind of `ErrorView` like can be seen in the `GradesScreen` etc.